util/src/main/java/org/folio/metastorage/util/SubDocument.java

```diff
@@ -0,0 +1,66 @@
+package org.folio.metastorage.util;
+
+import io.vertx.core.buffer.Buffer;
+import javax.xml.stream.XMLStreamConstants;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+
+public final class SubDocument {
```

Comment:
Maybe rename to XmlSerializer since this class allows serialisation at any level in the tree.

Reply:
Done